### DISCUSSION TITLE
Welcome to PPS-MH8A-Transmitter Discussions!

### DISCUSSION THREAD

**Discussion by rg422** [rg422/PPS-MH8A-Transmitter#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by TheCryingMonkey** [comment]
I made an account just to comment. After 8 hours of manually inputting sequential serial numbers into my dive app due to a missing serial number from one of my transmitters, I finally found this project. The ONLY thing on the internet talking about how to find an MH8A serial number. 
With only a pair of headphones, a desktop DAC, and Audacity, holy sh*t this worked first time...I'm speachless. 
I have zero experience in this field.
THANK YOU, you just saved me days of effort or £250.

**Reply by rg422** [reply]
Awesome! I'm glad it could be of use
